### DISCUSSION TITLE
right or wrong

### DISCUSSION THREAD

**Discussion by Mtigee** [Ma7moudTiger/Grow#22] [Q&A] [answered]
![image](https://github.com/Ma7moudTiger/Grow/assets/156967253/26712acb-b992-4755-a3e3-6f7cde989231)

**Comment by mtiger114** [comment, marked as answer]
Wrong .. try this one
![image](https://github.com/Ma7moudTiger/Grow/assets/105823344/6b949622-a323-4a11-b102-f0d3c84a52cd)
